en-US/mozorg/technology.lang

```diff
@@ -0,0 +1,116 @@
+## Note: Demo server is available at https://www-dev.allizom.org/technology
+## URL: https://www-dev.allizom.org/%LOCALE%/technology
+
+;We’re opening up a whole new Internet.
+We’re opening up a whole new Internet.
+
+
+;Explore some of our latest innovations — built with open Web technologies and designed to help keep the Internet healthy and accessible forever.
+Explore some of our latest innovations — built with open Web technologies and designed to help keep the Internet healthy and accessible forever.
+
+
+# This  string is a potential navigation label
+;Innovations
+Innovations
+
+
+# This  string is a potential navigation label
+;Web Innovations
```

Comment:
Same here, this goes into main.lang.

Add in the note that this is currently not used in the menu, it's an alternative text for Innovations.